### PR TITLE
allow auth.username and auth.audit_name in script values and descriptions

### PR DESCRIPTION
**Pull request by autogun**
This an attempt to support https://github.com/bugy/script-server/issues/217.
I also thought having those variables templated in script's config `description` field aswell.

### CODE REVIEW

**Comment by bugy** [comment]
Hi @autogun thanks for your changes!
Why did you decide to use "resolve_default" instead of "replace_auth_vars"? To be honest, I think now the naming is a bit confusing

**Comment by autogun** [comment]
Hey @bugy,

You mean as simple as `self.description = replace_auth_vars(config.get('description'))`? 😄

**Comment by bugy** [comment]
Exactly (you would also need to path auth arguments). And the same for the script value

File: src/model/model_helper.py [hunk 1]
```diff
@@ -189,7 +189,9 @@ def fill_parameter_values(parameter_configs, template, values):
 
 def replace_auth_vars(text, username, audit_name):
     result = text
-
+    if not result:
+        return
```
Comment (bugy) [inline src/model/model_helper.py:193]: Please `return result`

**Comment by bugy** [comment]
Thanks!

**Pull request merged**